network/network.html

```diff
@@ -0,0 +1,99 @@
+ <head>
+    <title>Wifi Network</title>
+    <meta charset="utf-8">
+    <link href="../base1/cockpit.css" type="text/css" rel="stylesheet">
+    <script src="../base1/jquery.js"></script>
+    <script src="../base1/cockpit.js"></script>
+</head>
+<body>
+	<button class="network"  id="1" style="margin-left: 20px; margin-top: 20px;"> <font size="2" face="sans-serif"> Change SSID </font></button>
+	<button class="network" id="2" style="margin-top:20px;"> <font size="2" face="sans-serif">Reset to Default </font></button>
+	<div class="1" id="div1" style='max-width: 400px; margin-left:20px; margin-top: 20px'>
+	<table class="cockpit-form-table">
+            <tr>
+                <td><label class="control-label" for="new_ssid"> <font size="2" face="sans-serif">New SSID </font></label></td>
+                <td><input class="form-control" id="new_ssid" value=""></td>
+            </tr>
+            <tr>
+                <td><label class="control-label" for="password"> <font size="2" face="sans-serif">Password</label></td>
+                <td><input class="form-control" id="password" value=""></td>
+            </tr>
+        </table>
+	<button class="btn btn-default btn-primary" id="change_ssid" style="margin-top: 20px; margin-left:100px; auto;'> <font size="2" face="sans-serif">Rename SSID </font></button>
+        <span id="result"></span>
```

Comment:
watch out for tabs and spaces. You have a mix of both.   Use 4 spaces for indent.